src/Field/Field.tsx

```diff
@@ -39,11 +39,17 @@ export const Field = ({
       {label && (
         <Box mb={{ custom: outlined ? 4 : 0 }}>
           {renderAsTitle ? (
-            <Title htmlFor={id}>{label}</Title>
+            <Text tag="label" typo="heading-small" htmlFor={id} mb="16px">
+              {label}
+            </Text>
           ) : (
-            <Text tag="label" color="subtext" typo="label" htmlFor={id}>
+            <Text tag="label" typo="label" color="subtext" htmlFor={id}>
```

Comment:
We don't need any margin bottom for this case?

Reply:
I've just replaced old typos with the new one design hasn't changed so if it was fine before should be fine now